test/server/compiler-test.js

```diff
@@ -363,12 +363,8 @@ describe('Compiler', function () {
         it('Should have definitions for all TestController methods', async function () {
```

Comment:
Should not merge until docs are ready (a comment to test the "Require conversations to be resolved" Github feature.